### PR TITLE
(#3385) NCIDS for Application Pages

### LINKED ISSUE #3385: Story: Update Application Pages for NCIDS Layout
## As a user, I would like to view Application Page content using the NCIDS, so that I have a better navigation experience.

This ticket is to update the application page templates which use both a full-page layout without left navigation and a layout with left navigation.

As part of this ticket, the Inner Pages background color should be updated to be #FFFFFF

> **ESTIMATE** 8

### Acceptance Criteria

```gherkin
Scenario: [title]
  Given [context]
    And [some more context]
  When  [event]
  Then  [outcome]
    And [another outcome]
```

### Resources:
* Feature: JIRA Ticket #
* Mockups: *Here goes a URL to or the name of the mockup(s) in inVision*
* Web Page: *Here goes a URL to an existing web page*

### Notes

*Some complementary notes if necessary:*

* > Here goes a quote from an email
* Here goes whatever useful information can exist…

---

### Prerequisites
- *required dependency for identifying the problem, scope and then solution for this issue*
- [ ] #3240 

### Tasks
- [ ] templates/content/application_page/page--cgov-application-page--no-left-nav.html.twig
  - https://www.cancer.gov/search/results?swKeyword=cancer
  - https://www.cancer.gov/about-cancer/treatment/clinical-trials/search
  - https://www.cancer.gov/research/resources
- [ ] templates/content/application_page/page--cgov-application-page--full.html.twig
  - https://www.cancer.gov/publications/dictionaries/cancer-terms
  - https://www.cancer.gov/publications/dictionaries/cancer-drug
  - https://www.cancer.gov/publications/dictionaries/genetics-dictionary

- [ ] update all app modules (new task?)
- [ ] templates/app_module/r4r/app-module--cgov-r4r-app.html.twig
- [ ] templates/layout/html--cgov-application-page.html.twig:
   - [ ] replace skipnav with `.usa-skipnav` 

#### Node templates
- [ ] templates/content/application_page/node--cgov-application-page--full.html.twig

### Technical Notes
- CTS should be moved to be a generic app module by the time we get here.

### LINKED ISSUE #3522: Update CTS LiveHelp pop-up for NCIDS Layouts
### Issue description
The Proactive Chat pop-up is currently missing styling on the inner and application pages on which it shows. The pop-up styles need to be updated to work in NCIDS.

This is related to #3385 and #3379 and maybe #3380 

> **ESTIMATE** TBD

### Steps to reproduce the issue

1.  
2. 
3. 

### What's the expected result?
- The LiveHelp pop-up displays with it's current CGov styling.

### What's the actual result?
- The LiveHelp pop-up is smushed into the bottom left corner, with no styling.

### Additional details / screenshot
livehelp

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by bryanpizzillo**
ODE: http://ncigovcdode479.prod.acquia-sites.com
Closes #3385
Closes #3522

Note: The R4R pages will still have the external link icon for non-external links when viewed on an ODE or locally due to the way the regex statement is formatted. The regex statement checks if the URL is a .gov, hence it will not work on our .com ODE.

See https://github.com/NCIOCPL/cgov-digital-platform/pull/3470 for previous notes

### CODE REVIEW

**Comment by bennettcc** [comment]
Two items to fix per Fed discussion on 10/4 @dev-rana-publicis @olitharp-nci @bryanpizzillo:

1.	Dynamic/Manual Listings: reduce padding between "locations" label and descriptive text to indicate which listing they go with 
Prod: https://www.cancer.gov/about-cancer/treatment/clinical-trials/disease/breast-cancer/ 
ODE: https://ncigovcdode479.prod.acquia-sites.com/about-cancer/treatment/clinical-trials/disease/breast-cancer/ 

2.	Search Results: Add space in the item per page selector styling so the number and arrow keys are not overlapping (the number valune and arrows are squished in all browsers)
Prod: https://www.cancer.gov/search/results?swKeyword=cancer 
ODE: https://ncigovcdode479.prod.acquia-sites.com/search/results?swKeyword=cancer

**Review by sarinapadilla — APPROVED** [review]
This PR passes internal product review!

**Comment by bennettcc** [comment]
ODDC Feds have approved the app modules during the 10/11 PM check in - please merge in!

@olitharp-nci @dev-rana-publicis @bryanpizzillo

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/entrypoints/app-module/app-module-legacy.scss [hunk 1]
```diff
@@ -0,0 +1,111 @@
+#NCI-glossary-app-root,
```
[the remark this answers is not in the document]
Comment (bryanpizzillo) [reply docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/entrypoints/app-module/app-module-legacy.scss:1]: like
`The spacing between pager items was off, this styling tightens it up.`

**Review by bryanpizzillo — COMMENTED** [review]
So I cannot request changes or approve my own PR, but @dev-rana-publicis I have 2 things for your commits. All my code appears to be perfection, if I don't say so myself. (Of course that is why I should not review my own PR.) Adding @arcepaul as a review for my stuff.

**Review by arcepaul — APPROVED** [review]

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/src/libraries/liveHelpPopup/ProactiveLiveHelp.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ export default class ProactiveLiveHelp {
         popupElement.id = this.options.popupID;
         popupElement.classList.add('ProactiveLiveHelpPrompt');
         popupElement.innerHTML = popupMarkup;
-        document.querySelector('body').appendChild(popupElement);
+        document.querySelector('#nci-modal-area').appendChild(popupElement);
```
[the remark this answers is not in the document]
Comment (arcepaul) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/src/libraries/liveHelpPopup/ProactiveLiveHelp.js:40]: @dev-rana-publicis ^bump

[the remark this answers is not in the document]
Comment (dev-rana-publicis) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/src/libraries/liveHelpPopup/ProactiveLiveHelp.js:40]: It is not new, already pre-existent 
https://github.com/NCIOCPL/cgov-digital-platform/commit/ffee14827531191f0d88f1e2c7c45f0cc520553a#diff-d8bf45a1f521b91a71cebdddaf22aae03e44acfb6939e279c5fdf18d8b1d9a1bR87

**Pull request merged**